### ISSUE TITLE
Build error app-debug.apk does not exist for add2app under flutter stable channel

### ISSUE THREAD

**Issue by burnermanx** [flutter/flutter#121256]
## Steps to Reproduce

1. Create an add-to-app project
2.  Try to run on Android Studio or execute `flutter run`

**Expected results:** 
Project runs on emulator

**Actual results:** 
Build error. Failed to install APK, because `build/host/apk/app-debug.apk` does not exist.

**Flutter Doctor:**

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.5-0.0.pre.1, on Ubuntu 22.04.2 LTS
    5.19.0-32-generic, locale pt_BR.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[!] Android Studio (version 2022.1)
    ✗ Unable to find bundled Java version.
[✓] VS Code
[✓] Connected device (3 available)
[✓] HTTP Host Availability

! Doctor found issues in 1 category.

```

Cherry-picking 5dabe102a2581a134dbf10ffc7fc8458d9bd1c05 fixes this problem.

**Comment by huycozy** [comment]
Thanks for the report. This issue seems to have been fixed on the latest master channel.

I'll be closing this issue since it has already been fixed on the latest master channel. If you still experience this issue, kindly leave a comment below and I will reopen this. Thank you!

**Comment by burnermanx** [comment]
> Thanks for the report. This issue seems to have been fixed on the latest master channel.
> 
> I'll be closing this issue since it has already been fixed on the latest master channel. If you still experience this issue, kindly leave a comment below and I will reopen this. Thank you!

It's possible to cherry-pick this fix to stable channel? On stable version (3.7.5 actually) I still have this problem

**Comment by huycozy** [comment]
You can read more about the build release channels here https://github.com/flutter/flutter/wiki/Flutter-build-release-channels. And also take a look at these documents for more information:

- https://github.com/flutter/flutter/wiki/Flutter-build-release-channels#will-a-particular-bug-fix-be-provided-in-a-hotfix-release
- https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process
